core/templates/dev/head/css/oppia.css

```diff
@@ -1026,8 +1089,8 @@ md-card.oppia-gallery-tile {
 }
 
 .oppia-html-select .oppia-html-select-option oppia-noninteractive-image {
```

Comment:
I've reinstated it, but are you sure they're not supposed to be deleted? All these changes were taken when I merged develop with upstream...